diff --git a/lib/webpackUtils.js b/lib/webpackUtils.js
index 369b436ff3570de963e840e46016df8543f810da..914e7aeb98a50b51eccfc3d1172da5bbf90cffdc 100644
--- a/lib/webpackUtils.js
+++ b/lib/webpackUtils.js
@@ -28,7 +28,7 @@ function getWebpackLoaderCompilerName(context) {
 function createFileLoaderUtils({ isServer, }) {
     // Files/images < urlLoaderLimit will be inlined as base64 strings directly in
     // the html
-    const urlLoaderLimit = constants_1.WEBPACK_URL_LOADER_LIMIT;
+    const urlLoaderLimit = 0;
     const fileLoaderFileName = (folder) => path_1.default.posix.join(constants_1.OUTPUT_STATIC_ASSETS_DIR_NAME, folder, '[name]-[contenthash].[ext]');
     const loaders = {
         file: (options) => ({
